flink-runtime/src/main/java/org/apache/flink/streaming/runtime/tasks/mailbox/TaskMailboxImpl.java

```diff
@@ -96,37 +96,26 @@ public boolean hasMail() {
         return !batch.isEmpty() || hasNewMail;
     }
 
-    @Override
-    public int size() {
-        final ReentrantLock lock = this.lock;
-        lock.lock();
-        try {
-            return batch.size() + queue.size();
-        } finally {
-            lock.unlock();
-        }
-    }
-
     @Override
     public Optional<Mail> tryTake(int priority) {
         checkIsMailboxThread();
         checkTakeStateConditions();
-        Mail head = takeOrNull(batch, priority);
-        if (head != null) {
-            return Optional.of(head);
+        Mail headEmail = takeOrNull(batch, priority);
+        if (headEmail != null) {
+            return Optional.of(headEmail);
```

Comment:
`head` -> `headEmail`. The latter is a better variable name. Some code uses `headEmail` and other uses `head` before this PR. We should unify the name.